ios/RNSentry.m

```diff
@@ -291,6 +291,33 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
     resolve(@YES);
 }
 
+RCT_EXPORT_METHOD(captureScreenshot: (RCTPromiseResolveBlock)resolve
```

Comment:
M: Can we actually reuse this? https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/SentryScreenshot.m

Reply:
I think we could expose `takeScreenshots` using the PrivateSDKs header.